ts/webui/src/static/const.ts

```diff
@@ -6,7 +6,7 @@ const METRIC_GROUP_UPDATE_SIZE = 20;
 
 const prefix = getPrefix();
 
-const MANAGER_IP = prefix === undefined ? '/api/v1/nni' : `${prefix}`;
+const MANAGER_IP = prefix === undefined ? '/api/v1/nni' : `api/v1/nni${prefix}`;
```

Comment:
prefix should be before `/api`?

Reply:
But when rest api is `prefix/api/v1/nni`, it still is conflict with ui router. because they are in the same directory `prefix/xxx` .

Reply:
![image](https://user-images.githubusercontent.com/61399850/119626816-2a307280-be3e-11eb-8bf1-8c4ed7a7e3db.png)
test with ui router `ip:port/api/oview` and rest api `ip:port/api/v1/nni/trial-jobs`, webportal is normal!

Reply:
Hi @liuzhe-lz , do you have any concern?

Reply:
Let's make it work first.